src/tito/builder/main.py

```diff
@@ -614,7 +614,7 @@ def tgz(self):
         debug("Scanning for sources.")
         cmd = "/usr/bin/spectool --list-files '%s' | awk '{print $2}' |xargs -l1 --no-run-if-empty basename " % self.spec_file
         result = run_command(cmd)
-        self.sources = map(lambda x: os.path.join(self.rpmbuild_gitcopy, x), result.split("\n"))
+        self.sources = list(map(lambda x: os.path.join(self.rpmbuild_gitcopy, x), result.split("\n")))
```

Comment:
Surely could, I was going after the smallest change. But if you want more readable code, that certainly can be done ;)